datastore/mockstore/mockstore.go

```diff
@@ -61,7 +61,10 @@ func (c mockCollection) UpsertId(selector interface{}, update interface{}) (*mgo
 }
 
 func (c mockCollection) UpdateId(selector, update interface{}) error {
-	c.Called(selector, update)
+	res := c.Called(selector, update)
+	if len(res) > 0 {
```

Comment:
is this check necessary?

Reply:
yes, res is an array, if I try to do `res.Error(0)` and it's an empty array it panics

Reply:
Can you also fix https://github.com/kubeapps/common/blob/1fdcd115f9191ac08187a24f760457ac49c64fd0/datastore/mockstore/mockstore.go#L117 then? Also, let's use `args` as the var name here instead of `res`.